README.md

```diff
@@ -27,7 +27,7 @@ As of February 2020, it is in a very early stage of development.
 Important links:
 
 - **[Documentation][docs]**
-- **[Org-roam Slack][slack]**
+- **[Org-roam Discourse][discourse]**
```

Comment:
I don't think we should be removing the Slack.  It's still useful for live support.

Reply:
Ok, I thought new users shouldn't be directed there. I'll fix.